core/domain/config_domain_test.py

```diff
@@ -84,24 +94,33 @@ def test_to_dict(self):
 class ConfigPropertyRegistryTests(test_utils.GenericTestBase):
     """Tests for the config property registry."""
 
-    def test_config_property_schemas_are_valid(self):
+    def test_config_property_schemas_are_valid(self) -> None:
         for property_name in (
                 config_domain.Registry.get_all_config_property_names()):
-            schema = config_domain.Registry.get_config_property(
+            schema = config_domain.Registry.get_config_property( # type: ignore[union-attr]
```

Comment:
Why is this ignore needed?

Reply:
as we made `get_config_property` to return optional[configproperty], then mypy assumes returned value by  get_config_property does not contain schema property.

Reply:
Then you can do
```python
config_property = config_domain.Registry.get_config_property(property_name)
# Ruling out the possibility of None for mypy type checking.
assert config_property is not None
schema_utils_test.validate_schema(config_property.schema)
```

Reply:
Done!